packages/flutter_tools/lib/src/isolated/resident_web_runner.dart

```diff
@@ -72,25 +82,43 @@ const String kExitMessage = 'Failed to establish connection with the application
   'instance in Chrome.\nThis can happen if the websocket connection used by the '
   'web tooling is unable to correctly establish a connection, for example due to a firewall.';
 
-/// A hot-runner which handles browser specific delegation.
-abstract class ResidentWebRunner extends ResidentRunner {
```

Comment:
Originally there was an abstract class and 2 different implementations, which was used to switch on/off dwds before it was the default. Since there is only one impl now, just combine the class into the interface